models/project/project.go

```diff
@@ -39,7 +39,7 @@ func NewProject(id int64, userID int64, title string, description string, reposi
 	return project
 }
 
-func FindProject(projectID int64) *ProjectStruct {
+func FindProject(projectID int64) (*ProjectStruct, error) {
```

Comment:
__[golint]__
* [golint] exported function FindProject should have comment or be unexported